glommio/src/io/buffered_file.rs

```diff
@@ -122,7 +128,7 @@ impl BufferedFile {
         enhanced_try!(source.collect_rw().await, "Writing", self.file)
     }
 
-    /// Reads data at the specified position into the user-provided buffer `buf`.
+    /// Reads data at the specified position into a buffer allocated by the OS.
```

Comment:
technically it is not the OS that allocates the buffer.

You are right that the previous comment was wrong (bad copy pasta from write, probably), but the comment refers to the `Vec<u8>` buffer that this returns. That one is allocated by us, not the OS.

Reply:
Ah, thanks. In that case, "allocated by this library" would perhaps be more appropriate?

Reply:
yes